### ISSUE TITLE
0.27 Does not generate equality

### ISSUE THREAD

**Issue by JoeyEremondi** [ott-lang/ott#21]
Minimal example:

```
grammar

  Kind, K :: kind_ ::= {{ coq-equality }}
    | KindStar :: :: KindStar
    | K -> K' :: :: ArrowKind

```

In ott 0.25, I get this output from `ott minimalTest.ott -o minimalTest.v`:

```coq
(* generated by Ott 0.25 from: minimalTest.ott *)

Require Import Arith.
Require Import Bool.
Require Import List.

Inductive Kind : Set := 
 | kind_KindStar : Kind
 | kind_ArrowKind (K:Kind) (K':Kind).
Lemma eq_Kind: forall (x y : Kind), {x = y} + {x <> y}.
Proof.
  decide equality; auto with ott_coq_equality arith.
Defined.
Hint Resolve eq_Kind : ott_coq_equality.
```

In 0.27, however, the equality is missing:

```coq
(* generated by Ott 0.27 from: minimalTest.ott *)

Require Import Arith.
Require Import Bool.
Require Import List.

Inductive Kind : Set := 
 | kind_KindStar : Kind
 | kind_ArrowKind (K:Kind) (K':Kind).


```

**Comment by palmskog** [comment]
Actually, this only applies to non-metavars, and should be fixed by #22.

**Comment by palmskog** [comment]
Since #22 was merged, I'm closing this. If the problem returns in some form, please re-open.
